app/controllers/hiring_staff/organisations/user_preference_controller.rb

```diff
@@ -0,0 +1,38 @@
+class HiringStaff::Organisations::UserPreferenceController < HiringStaff::BaseController
+  before_action :verify_school_group
+
+  def show
+    @user_preference_form = UserPreferenceForm.new(current_user, current_organisation)
+  end
+
+  def update
+    @user_preference_form = UserPreferenceForm.new(current_user, current_organisation, user_preferences_form_params)
+
+    if @user_preference_form.valid?
+      @user_preference_form.save
+      redirect_to organisation_path
+    else
+      render :show
+    end
+  end
+
+  private
+
+  def update_managed_organisations
+    if params[:commit] == I18n.t('buttons.skip_this_step')
+      params[:user_preference_form][:managed_organisations] = ['all']
+    elsif params[:user_preference_form][:managed_organisations].blank?
+      params[:user_preference_form][:managed_organisations] = []
+    end
+  end
+
+  def user_preferences_form_params
```

Comment:
V minor:
`@user_preference_form` with singular preference but `user_preferences_form_params` with plural preferences

Reply:
Thank you kind friend